readme.md

```diff
@@ -26,7 +26,7 @@ dotProp.get({foo: {'dot.dot': 'unicorn'}}, 'foo.dot\\.dot');
 //=> 'unicorn'
 
 // setter
-var obj = {foo: {bar: 'a'}};
+let obj = {foo: {bar: 'a'}};
```

Comment:
Shouldn't this be `const`? You're not changing the binding of `obj`, just its contents.